### PR TITLE
Add Python rdp (Ramer-Douglas-Peucker algorithm in pure python)

### PR DESCRIPTION
**Pull request by gonzalocasas**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by gonzalocasas** [comment]
This is ready to review.

@conda-forge-admin, please ping team

File: recipes/rdp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - numpy
+    - python >=2.7
```
Comment (ocefpaf) [inline recipes/rdp/meta.yaml:24]: You should use the same constraint on host and run but note that we no longer support 2.7-3.5 so, even though the package may work on 2.7 we recommend you to use >=3.6

Comment (gonzalocasas) [reply recipes/rdp/meta.yaml]: Fixed on https://github.com/conda-forge/staged-recipes/pull/14404/commits/e322388b16c566c5c407ea26af3a0e8281c81a99!

**Comment by gonzalocasas** [comment]
@ocefpaf better like this?

**Pull request merged**